component/src/test/java/com/synopsys/integration/alert/component/authentication/web/SamlMetadataFileUploadTest.java

```diff
@@ -51,7 +53,7 @@ public void cleanResource() throws Exception {
     @Test
     public void performUpload() {
         SamlMetaDataFileUpload action = new SamlMetaDataFileUpload(descriptorKey, authorizationManager, filePersistenceUtil);
-        Mockito.when(authorizationManager.hasUploadWritePermission(Mockito.anyString(), Mockito.anyString())).thenReturn(Boolean.TRUE);
+        Mockito.when(authorizationManager.hasUploadWritePermission(Mockito.any(ConfigContextEnum.class), Mockito.any(DescriptorKey.class))).thenReturn(Boolean.TRUE);
```

Comment:
We cannot use `Mockito .any()` here because it clashes with the deprecated `String` version of the method.